README.MD

```diff
@@ -0,0 +1,326 @@
+BrandedToken.js
+============
+
+BrandedToken.js supports interaction with BrandedToken-contracts.
+The steps below describe the process of staking an EIP-20 token to create a BrandedToken on the value chain (assumed to be ethereum ) and minting (creating) a value-backed Utility Branded Token on the sidechain for use in applications.
+
+#### Overview of different components:
```

Comment:
Adjusted, please review

Reply:
Thanks! Because the top-most heading, "BrandedToken.js" is higher than the other headings, please add a "#" to each of the headings (to make them all H2 or lower).

Relatedly, further down in the file, please add a space after the hashes in:
> "##Approve GatewayComposer..."

Reply:
^^^ For whoever is working on it, @abhayks1 or @achirala.